config/config-builder/src/full_node_config.rs

```diff
@@ -218,6 +221,8 @@ impl FullNodeConfig {
                 .full_node_networks
                 .get_mut(0)
                 .ok_or(Error::MissingFullNodeNetwork)?;
+            // @TODO: We need to determine a name for this Private network
```

Comment:
this is config-builder... if this is a validator node or a fullnode connected to a validator, then this should probably be "vfn"
otherwise this is the public network

let's just call all the networks in config-builder private "vfn" and let @mgorven help us adjust if possible.

I think @agclement has also spent time here, perhaps he can make a suggestion?

Reply:
That sounds good enough for now.